conference/fares.py

```diff
@@ -116,7 +117,7 @@ def pre_create_typical_fares_for_conference(conference, vat_rate,
                                             print_output=False):
     fares = []
 
-    for fare_code in AVAILABLE_FARE_CODES.keys():
+    for fare_code in list(AVAILABLE_FARE_CODES.keys()):
```

Comment:
Unnecessary `list`.

Reply:
For future reference: find . -type f -name \*.py -exec gawk -i inplace '{print gensub(/for (.+) in list\((.+)\):/, "for \\1 in \\2:", "g");}' {} \;